README.md

```diff
@@ -77,7 +77,8 @@ Where:
      Max requests per connection (default: 4294937295).
 
    --max-active-requests <uint32_t>
-     Max active requests (default: 4294937295).
+     Max active requests (default: 1). H2-only, the number of concurrent 
```

Comment:
Do you think this might distort common case results? I think the principle that should govern defaults is "would someone using NH out-of-the-box get reasonable results if performing common case benchmarking?". From that perspective, I'd be interested in hearing what you reckon.

Reply:
(pushed a change to implement defaulting to `100` for `max-active-requests` )

Reply:
In this case, we probably want to shoot for "what would be reasonable in a performance benchmarking scenario".